init_query.sql

```diff
@@ -14,7 +14,7 @@ SELECT
         WHEN sound.cldf_cltsReference like '%stop%' then 'stop' else 'dunno'
         END sound_class,
 	-- standardize speech rate
-	-- (utt.log_speech_rate - avg(utt.log_speech_rate) over()) / stdev(utt.log_speech_rate) over() as z_speech_rate,
+	(utt.log_speech_rate - z.avg_speech_rate) / z.speech_rate AS z_speech_rate,
```

Comment:
@Tarotis we can pull average and stdev (computed over all utterances) from the joined query `z`, see below.